src/main/kotlin/no/nav/familie/ef/mottak/integration/SaksbehandlingClient.kt

```diff
@@ -24,9 +27,23 @@ class SaksbehandlingClient(
 
     fun kanOppretteFørstegangsbehandling(personIdent: String, stønadType: StønadType): Boolean {
         val uriComponentsBuilder = UriComponentsBuilder.fromUri(uri)
-            .pathSegment("api/ekstern/behandling/kan-opprette-forstegangsbehandling")
+            .pathSegment("/api/ekstern/automatisk-journalforing/kan-opprette-forstegangsbehandling")
             .queryParam("type", stønadType.name)
         val response = postForEntity<Ressurs<Boolean>>(uriComponentsBuilder.build().toUri(), PersonIdent(personIdent))
         return response.data ?: error("Kall mot ef-sak feilet melding=${response.melding}")
     }
+
+    fun lagFørstegangsbehandlingOgBehandleSakOppgave(
```

Comment:
Et vesentlig poeng her er jo også at den skal ferdigstille journalføringen 🤔  Blir et langt navn for en funksjon, kanskje?

Reply:
`ferdigstillJournalføringOgOpprettFørstegangsbehandling` ?

Reply:
👍 
Reverterer tilbake til automatiskJournalfør